observability-test/spanner.ts

```diff
@@ -542,54 +541,52 @@ describe('ObservabilityOptions injection and propagation', async () => {
       db.formattedName_,
     );
 
-    it('run', done => {
-      database.getTransaction((err, tx) => {
-        assert.ifError(err);
-
-        tx!.run('SELECT 1', async () => {
-          tx!.end();
-
-          await tracerProvider.forceFlush();
-          traceExporter.forceFlush();
-
-          const spans = traceExporter.getFinishedSpans();
-          withAllSpansHaveDBName(spans);
-
-          const actualSpanNames: string[] = [];
-          const actualEventNames: string[] = [];
-          spans.forEach(span => {
-            actualSpanNames.push(span.name);
-            span.events.forEach(event => {
-              actualEventNames.push(event.name);
-            });
+    it('run', async () => {
+      let txn;
+      try {
+        [txn] = await database.getTransaction();
+        await txn.run('SELECT 1');
```

Comment:
Just curious, was there any particular reason for changing this test from callback to async/await. Was something breaking ?